package.json

```diff
@@ -112,7 +112,9 @@
     "react-helmet": "5.2.0",
     "react-live": "1.7.1",
     "react-router-dom": "4.2.2",
+    "react-sticky": "6.0.1",
```

Comment:
While looking into using this on my branch, I noticed that it is no longer maintained. Is this really the best package to do this, then?